ksql-engine/src/main/java/io/confluent/ksql/analyzer/StaticQueryValidator.java

```diff
@@ -89,6 +91,12 @@
       Rule.of(
           analysis -> !analysis.getLimitClause().isPresent(),
           "Static queries don't support LIMIT clauses."
+      ),
+      Rule.of(
+          analysis -> analysis.getSelectColumnRefs().stream()
+                  .map(ColumnRef::name)
+                  .noneMatch(n -> n.equals(SchemaUtil.ROWTIME_NAME)),
+          "Static queries don't support ROWTIME in the projection."
```

Comment:
Will the user know what "projection" means here? Would it be simpler to say something like "Static queries can't select the ROWTIME column?"

Reply:
'projection' is the standard db term for the select columns.  But... you may be right!